### ISSUE TITLE
Submit your first PR by solving a Leetcode problem

### ISSUE THREAD

**Issue by GouravRusiya30** [codedecks-in/LeetCode-Solutions#194]
- Go through the README file
- Check the Contributing guidelines
- Enjoy the open source experience

**Comment by pakosel** [comment]
Hey, can I contribute using C# ?

**Comment by febriaricandra** [comment]
nah i want to contribute using c# too..

**Comment by adamfadl** [comment]
c++

**Comment by keshav-rockwinder** [comment]
hey i want to contribute through c++ can you please specify and assign me this task?

**Comment by Appy-007** [comment]
@GouravRusiya30 I can contribute  using C program

**Comment by CodeexpartPrakashGithub** [comment]
hello,  i want to contribute using CPP .

**Comment by hellovaibhav** [comment]
i want to contribut using cpp

**Comment by pratyushkr11** [comment]
I want to contribute using cpp.

**Comment by hadeelq3** [comment]
I want to contribute using c++

**Comment by mikemostone** [comment]
Can I contribute using C++ or Python? An easier solution for my first Open Source Contribution for school

**Comment by GouravRusiya30** [comment]
Yes, people can contribute to this repo for `[hacktoberfest-2023](https://github.com/codedecks-in/LeetCode-Solutions/labels/hacktoberfest-2023)`
But please make sure to check other PRs to get the idea on how to submit the correct PR.

**Comment by nancy-singh10** [comment]
i want to contribute on this using c++ under hacktoberfest 2023

**Comment by Tejoooo** [comment]
heyy can u assign me this issue

**Comment by aryan-rohit** [comment]
@mikemostone assign this to me

**Comment by c99SRS** [comment]
@GouravRusiya30 can you please merge my PR
